tests/src/test/scala/system/basic/WskBasicTests.scala

```diff
@@ -75,6 +75,40 @@ class WskBasicTests
             stderr should include(errormsg)
     }
 
+    it should "accept a 3 part Fully Qualified Name without a leading '/'" in {
+        val auth: Seq[String] = Seq("--auth", wskprops.authKey)
+        val guestNamespace = wskprops.namespace
```

Comment:
this is going to be `_` is that ok?
is there a test where `_` is not used (and explicit namespace name is used? you can use `wsk.namespace.whois()` for that).